tests/test_culling.py

```diff
@@ -365,6 +366,8 @@ def test_reaper_shutdown_handler(
     created_hosts = db_get_hosts(created_host_ids)
     assert created_hosts.count() == host_count
 
+    event_producer_mock.write_event = mock.Mock()
```

Comment:
If the write_event method is replaced like this, then there is no need to use the _event_producer_mock_ fixture. It would be better if the _MockEventProducer_ class would record more than one call, but we probably don’t want to do that in this pull request. The call itself is however pretty useless and after your changes in this particular test case, it brings no value at all.

If you pass a plain ol’ _Mock_ instead of the whole _event_producer_, you’ll get _event_producer.write_event_ mock for free.

```suggestion
    event_producer_mock = mock.Mock()
```